### PR TITLE
Add anchor links for section headers

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: presto-docs/src/main/sphinx/themes/presto/static/presto.css_t [hunk 1]
```diff
@@ -26,12 +26,12 @@ body {
...
 }
 
-a:link, a:visited {
+a:link, a:visited, a.headerlink {
   font-weight: bold;
   text-decoration: none;
   color: #44af5c; }
 
-a:hover, a:active {
+a:hover, a:active, a.headerlink:hover {
```
Comment (findepi) [inline presto-docs/src/main/sphinx/themes/presto/static/presto.css_t]: redundant?

File: presto-docs/src/main/sphinx/conf.py [hunk 2]
```diff
@@ -94,6 +94,6 @@ def get_version():
 
 html_title = '%s %s Documentation' % (project, release)
 
-html_add_permalinks = ''
+html_add_permalinks = u'\u00A7'
```
Comment (findepi) [inline presto-docs/src/main/sphinx/conf.py]: although both junit & jdbi use `§` for this, i kind of would expect `#` 

anyway, they both show it on the left side of text (here it's on the right)

File: presto-docs/src/main/sphinx/themes/presto/static/presto.css_t (hunk 1 above)
Comment (electrum) [reply presto-docs/src/main/sphinx/themes/presto/static/presto.css_t]: No, there is a specific style for `a.headerlink:hover` in the included `haiku.css` (comes from the theme so no easy way to change it ... need to clean all that up sometime)

File: presto-docs/src/main/sphinx/conf.py (hunk 2 above)
Comment (electrum) [reply presto-docs/src/main/sphinx/conf.py]: Sounds good to me

**Pull request merged**